src/modules/ethereum/transactions/addressAction.ts

```diff
@@ -68,14 +68,41 @@ const toActionTx = (tx: IncomingTransaction, chain: Chain) =>
     nonce: -1,
   } as const);
 
+function isAsset(x: Asset | NFT): x is Asset {
+  return 'asset_code' in x && 'decimals' in x && 'symbol' in x;
+}
+function isNFT(x: Asset | NFT): x is NFT {
+  return 'contract_address' in x && 'token_id' in x;
+}
+
+function toNftAsset(x: NFT): NFTAsset {
```

Comment:
You can use this function as well
https://github.com/zeriontech/pulse-frontend/blob/master/src/features/send/SendForm/shared/convertNFTAsset.ts#L4